tmt/cli/__init__.py

```diff
@@ -0,0 +1,85 @@
+""" Command line interface for the Test Management Tool """
+
+import os
+from typing import Any, List
+
+import click
+
+import tmt
+import tmt.options
+import tmt.plugins
+import tmt.utils
+from tmt.cli.clean import clean
+from tmt.cli.click_context_object import ContextObject
+from tmt.cli.click_group import CustomGroup
+from tmt.cli.common_options import verbosity_options
+from tmt.cli.init import init
+from tmt.cli.lint import lint
+from tmt.cli.plans import plans
+from tmt.cli.run import run
+from tmt.cli.setup import setup
+from tmt.cli.status import status
+from tmt.cli.stories import stories
+from tmt.cli.tests import tests
```

Comment:
This global namespace pollution does not seem right to me. The only purpose of `tests` is to be used in `main.add_command(tests)` call at the end of the file. Could we use some kind of import magic?

```python
for name in ('run', 'tests', 'plans', ...):
  command = tmt.plugins.import_member(f'tmt.cli.{name}, name)
  main.add_command(command)
```

Reply:
I guess we could but I personally don't see nothing wrong with being explicit in importing. If You don't like potential name clashes, we could use absolute imports (but I prefer `from` imports but that's a matter of personal taste)